generate-component-docs/__snapshots__/contract.test.ts.snap

```diff
@@ -5991,6 +5991,22 @@ Object {
 }
 `;
 
+exports[`TextLinkButton 1`] = `
+Object {
+  exportType: component,
+  props: {
+    children: ReactNode
+    data?: Record<string, string | number>
+    hitArea?: 
+        | "large"
+        | "standard"
+    id?: string
+    onClick?: () => void
+    showVisited?: boolean
```

Comment:
Does this still make sense?

Reply:
Good catch 👍

Reply:
Again ignore this

Reply:
No, I think you're right on this. A button can't be marked as visited.